docs/SnapshotTesting.md

```diff
@@ -264,6 +270,8 @@ Now, every time the snapshot test case runs, `Date.now()` will return `148236336
 
 Always strive to use descriptive test and/or snapshot names for snapshots. The best names describe the expected snapshot content. This makes it easier for reviewers to verify the snapshots during review, and for anyone to know whether or not an outdated snapshot is the correct behavior before updating.
 
+:::
+
```

Comment:
```suggestion
```

Reply:
Thanks a lot for all comments.

If we make the global state supports the concurrent mode, it might solve the problem. I haven't found a way that makes the global state supports the concurrent mode.